packages/ui/src/lib/hooks/useTranslation/useTranslation.tsx

```diff
@@ -58,7 +58,7 @@ export const TranslationProvider = track(function TranslationProvider({
 		let isCancelled = false
 
 		async function loadTranslation() {
-			const localeString = locale ?? navigator.language.split(/[-_]/)[0]
+			const localeString = (locale ?? navigator.language).split(/[-_]/)[0]
```

Comment:
`navigator.language` will be something like `en-US` and so will `locale`. We want to get the language portion of either.

Reply:
See https://github.com/tldraw/tldraw/pull/1201/files#r1176852583